flask_appbuilder/templates/appbuilder/general/lib.html

```diff
@@ -20,16 +22,27 @@
                 {% endif %}
                 <input type="hidden" id="action" name="action" />
             </form>
-
-            <a href="javascript:void(0)" class="btn btn-sm btn-primary"
-               {% if action.confirmation %}
-               onclick="var a = new AdminActions(); return a.execute_single('{{path}}','{{action.confirmation}}');">
-               {% else %}
-               onclick="var a = new AdminActions(); return a.execute_single('{{path}}',false);">
-               {% endif %}
+            <a
+                id="btn-action-{{ endpoint }}"
+                href="javascript:void(0)"
```

Comment:
Unfortunately this also violates the inline script policy. Could we change that to an empty href or even make it a button? Same goes for every other a tag with `void(0)`